integration-tests/downscope/downscope_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 Google LLC.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package downscope
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"google.golang.org/api/option"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"golang.org/x/oauth2/google/downscope"
+	storage "google.golang.org/api/storage/v1"
+)
+
+const (
+	envServiceAccountFile = "GCLOUD_TESTS_GOLANG_KEY"
+	rootTokenScope        = "https://www.googleapis.com/auth/cloud-platform"
+)
+
+var (
+	object1 = "cab-first-c45wknuy.txt"
```

Comment:
These could be const as well.

Reply:
Thanks, just changed it.  A few of the comments you've made are because the structure of the code changed a few times and I forgot to update some legacy decisions that were made.